contrib/executor/init/pkg/runner/runner.go

```diff
@@ -81,7 +81,8 @@ func (r *InitRunner) Run(ctx context.Context, execution testkube.Execution) (res
 		}
 
 		shebang := "#!" + shell + "\nset -e\n"
-		entrypoint := shebang
+		// No set -e so that we can run the post-run script even if the command fails
+		entrypoint := "#!" + shell + "\n"
```

Comment:
Failure of pre-run script should rather also lead to failure, without running the test